.circleci/config.yml

```
@@ -44,6 +44,15 @@ jobs:
       - store_test_results:
           path: test_results
 
+  chromatic: 
+    executor: node-fermium-executor
+    steps:
+    - attach_workspace:
+        at: .
+    - run:
+        name: Chromatic Build and Deploy
+        command: npm run chromatic --project-token=79723ccb1e7b --exit-zero-on-changes
```

When running NPM script with arguments you need to add `--` before passing flags/arguments'. Also, this is where the env variable in CircleCI would come in

```suggestion
        command: npm run chromatic -- --project-token=$CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes
```

---

here as well, I added my token for testing and forgot to push the changes back. sorry again